deployer-service/main.go

```diff
@@ -0,0 +1,31 @@
+package main
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/NicolasLopes7/shipthing/config"
+)
+
+func main() {
+	err := config.InitConfig()
+
+	redisCtx := context.Background()
+
+	if err != nil {
+		panic(err)
+	}
+
+	for true {
+		res := config.RedisClient.BRPop(redisCtx, 0, "builds")
```

Comment:
`BRPOP` is a blocking list pop primitive. It is the blocking version of [RPOP](https://redis.io/commands/rpop) because it blocks the connection when there are no elements to pop from any of the given lists. An element is popped from the tail of the first list that is non-empty, with the given keys being checked in the order that they are given.